examples/roxy-test-project/backup-build.gradle

```diff
@@ -0,0 +1,20 @@
+buildscript {
```

Comment:
What is this file for? Looks like it was committed by mistake and should be deleted.